ow_plexil/src/plans/HealthMonitorDemo.plp

```diff
@@ -0,0 +1,61 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Demo unstows and stows the arm if the Camera is set as operable
+// from the HealthMonitor. If it is inoperable, and the camera is operable
+// it takes photos. If both the Camera and Arm are inoperable, it prints out an
+// error message and waits for them to be operable. 
+
+#include "plan-interface.h"
+
+LibraryAction HealthMonitor (InOut Boolean AllOperable,
+                             InOut Boolean ArmOperable,
+                             InOut Boolean AntennaOperable,
+                             InOut Boolean CameraOperable,
+                             InOut Boolean PowerOperable);
+
+HealthMonitorDemo: Concurrence
+{
+
+  Boolean AllGood;
+  Boolean ArmGood;
+  Boolean AntennaGood;
+  Boolean CameraGood;
+  Boolean PowerGood;
+  
+  log_info("Starting HealthMonitorDemo...");
+
+  Monitor:
+  {
+  LibraryCall HealthMonitor (AllOperable = AllGood,
+                             ArmOperable = ArmGood,
+                             AntennaOperable = AntennaGood,
+                             CameraOperable = CameraGood,
+                             PowerOperable = PowerGood);
+  }
+
+  DemoPattern:
+  {
+    Repeat true;
+    Wait 3;
+
+    if ArmGood{
+      Invariant ArmGood;
+      log_info("Unstowing and stowing arm...");
+      LibraryCall ArmUnstow();
+      LibraryCall ArmStow();
```

Comment:
I wonder if this block is susceptible to the "SynchronousCommand problem", but that's out of scope of this PR.

Reply:
I would as well, interestingly enough, I havent ran into the SynchronousCommand problem yet for this plan. Its strange because normally the ArmUnstow() and ArmStow() always have that issue.